### PR TITLE
Oie docs embedded revisions general 2

### PR DESCRIPTION
**Pull request by robertnicolo-okta**
## Description:
General style guide adherence updates.

### Resolves:

* [OKTA-406076](https://oktainc.atlassian.net/browse/OKTA-406076)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md [hunk 1]
```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
 
 For the sample app use the **Issuer URI** from the custom authorization server
 named “default”.  To find this value, go to **Security > API** in the Admin
-console. Under the **Authorization servers** tab, the default authorization
+Console. Under the **Authorization servers** tab, the default authorization
 server’s Issuer URI is displayed in the list.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:38]:
```suggestion
Server’s Issuer URI appears in the list.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md [hunk 2]
```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
...
 
 An example of the URI is:  `https://dev-foo.okta.com/oauth2/default`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:40]:
```suggestion
Example URI:  `https://dev-foo.okta.com/oauth2/default`.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md [hunk 3]
```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
...
 
 Note that there are two main types of authorization servers: Org and custom:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:42]:
```suggestion
> **Note:** There are two main types of authorization servers: Org and Custom
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md [hunk 4]
```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
...
 
-1. **Org**: If you want to use the Org authorization server use the following
+* **Org**: If you want to use the Org authorization server use the following
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:44]:
```suggestion
* **Org**: If you want to use the Org Authorization Server, use the following
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md [hunk 5]
```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
...
    url: `https://dev-foo.okta.com`.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:45]:
```suggestion
   URL: `https://dev-example.okta.com`.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md [hunk 6]
```diff
@@ -34,23 +34,23 @@ decide to also use the widget). The list of settings is shown below:
...
-1. **Custom**: There are two types of custom authorization servers:
-   1. **default**: The default authorization server is predefined and
+* **Custom**: There are two types of custom authorization servers:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:46]: this isn't necessarily accurate. There is the org auth server and then there are custom auth servers. The FIRST custom auth server is automatically created for them. Then, they can create their own additional custom auth servers. Saying that there are two different types of custom auth servers implies that there is a difference between the custom auth server ("default") that is automatically created for them in their org and the ones that they subsequently create after that. I feel that this wording is misleading.

**Review by susanharper-okta — COMMENTED** [review]
comments

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md (hunk 6 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md:46]: Will improve the clarity of the text in the next PR.

**Pull request merged**